html/opusjs/decoder.js

```diff
@@ -1,3 +1,18 @@
+addEventListener('error', (event) => {
```

Comment:
So, as you can probably tell, these are autogenerated files we got from elsewhere (they are output of emscripten). I believe that there is some kind of special asm.js magic detection. I don't know much about the details, but I have a vague impression that, to avoid breaking that, you can't edit above the `// EMSCRIPTEN_END_ASM` line.